azure-pipelines.yml

```diff
@@ -1,18 +1,53 @@
 pool:
   vmImage: ubuntu-latest
 
+variables:
+  - name: imageTag
+    value: ${{ replace(variables['Build.SourceBranch'],'refs/tags/','') }}
+  - name: isTagBuild
```

Comment:
This variable means that the commit is tagged, right? Then I would maybe rather called it `isTaggedBuild` or `isBuildTagged` just to be more explainable.